folio-spring-i18n/README.md

```diff
@@ -0,0 +1,103 @@
+# Documentation for folio-spring-i18n features
+
+- [Introduction](#introduction)
+- [Adding this module to your project](#adding-this-module-to-your-project)
+- [Using translations in your code](#using-translations-in-your-code)
+  - [`String format(String key, Object... args)`](#string-formatstring-key-object-args)
+    - [Example](#example)
+  - [`String formatList(Collection<?> list)`](#string-formatlistcollection-list)
+    - [Example](#example-1)
+
+## Introduction
+
+Translations may be performed in backend modules using this library, per [this TC decision](https://wiki.folio.org/x/SqTc). These translations work the same as UI modules:
+
+- Translations are stored in JSON files within the module's path, with a structure like `/translations/mod-foo/locale.json` (`/translations/` may be customized by altering the POM's mapping below, however, this is not recommended as Lokalise will not pick it up).
+- Each JSON file is named for a locale and region, e.g. `en_ca.json`.
+- The `en.json` file is the source of truth and fallback; other translation modules will be generated and tracked separately by Lokalise.
+- Translation keys are prepended with the module name, e.g. `mod-foo.title`.
+
+This library provides a number of features, including:
+
+- Support for LocalDate and LocalTime formatting (use of time zones/offsets are not supported, as there is no way for clients to provide this information)
+- Support for list formatting
+- Automatic fallbacks based on languages and server locale (`zh_CN` -> `zh` -> `en` -> server locale)
+- Full ICU token support
+
+See https://wiki.folio.org/display/I18N/How+to+Translate+FOLIO for more information on how translations are contributed and tracked.
+
+## Adding this module to your project
+
+To add this module to your project, add the following to your `pom.xml` in `<dependencies>`:
+
+```xml
+<dependency>
+  <groupId>org.folio</groupId>
+  <artifactId>folio-spring-i18n</artifactId>
+  <version>${folio-spring-base.version}</version>
+</dependency>
+```
+
+Then, add the following to your `pom.xml` in `<build>`:
+
+```xml
+  <resources>
+    <resource>
+      <directory>src/main/resources</directory>
+    </resource>
+    <resource>
+      <directory>${project.basedir}/translations</directory>
+      <targetPath>translations</targetPath>
+    </resource>
+  </resources>
+  <testResources>
+    <testResource>
+      <directory>src/test/resources</directory>
+    </testResource>
+    <testResource>
+      <directory>${project.basedir}/translations</directory>
+      <targetPath>translations</targetPath>
+    </testResource>
+  </testResources>
+```
+
+Now, create a folder `/translations/mod-name-here` and you're good to go!
```

Comment:
Why is it required to have a folder with the module name? Why it can't be just the translations folder?

Reply:
Following the same standard as UI modules.  This would also serve to prevent conflicts in cases where modules may share translations

Reply:
I believe Lokalise also expects this subdirectory